pkg/kube/client.go

```diff
@@ -714,8 +713,8 @@ func (c *Client) getSelectRelationPod(info *resource.Info, objPods map[string][]
 	}
 
 	for _, externalPod := range pods.Items {
-		pod := core.Pod{}
-		legacyscheme.Scheme.Convert(&externalPod, &pod, nil)
+		pod := v1.Pod{}
+		scheme.Scheme.Convert(&externalPod, &pod, nil)
```

Comment:
I don't think there is any reason to do any conversion here. `externalPod` is already of type v1.Pod.

Reply:
good catch, fixed